src/weather.js

```diff
@@ -0,0 +1,13 @@
+const weather = (cityName) => {
+  async function result() {
+    const response = await fetch(`http://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=873aa78430c524e7ef6505af11e25a57`, { mode: 'cors' });
```

Comment:
**[OPTIONAL]** 
* [ ] Kindly use a `try and catch` statement to handle your API request or any other means that would enable you to capture exceptions in a much cleaner way.